src/wasm/wasm-validator.cpp

```diff
@@ -1527,9 +1527,9 @@ void FunctionValidator::visitDataDrop(DataDrop* curr) {
 
 void FunctionValidator::visitMemoryCopy(MemoryCopy* curr) {
   shouldBeTrue(
-    getModule()->features.hasBulkMemory(),
+    getModule()->features.hasBulkMemoryOpt(),
     curr,
-    "Bulk memory operations require bulk memory [--enable-bulk-memory]");
+    "memory.copy operations require bulk memory [--enable-bulk-memory-opt]");
```

Comment:
Should the text also reflect " opt"?

Reply:
Actually, I'm not entirely sure what "opt" means...

Reply:
I have no idea... "option" or "optimization" both seem not quite right.

But how about "bulk memory opt" in the text?